creativecloud/blocks/nonprofit/nonprofit.js

```diff
@@ -19,9 +19,9 @@ const removeOptionElements = (element) => {
 
 // #region Constants
 
-const PERCENT_API_URL = 'https://sandbox-api.poweredbypercent.com/v1';
-const PERCENT_VALIDATION_API_URL = 'https://sandbox-validate.poweredbypercent.com/adobe-acrobat';
-const PERCENT_PUBLISHABLE_KEY = 'sandbox_pk_8b320cc4-5950-4263-a3ac-828c64f6e19b';
+const PERCENT_API_URL = 'https://api.goodstack.io/v1';
+const PERCENT_VALIDATION_API_URL = 'https://validate.goodstack.org/adobe-acrobat';
+const PERCENT_PUBLISHABLE_KEY = 'pk_ea675372-2eb2-4cf1-8b6a-358087bf8df5';
```

Comment:
Will keeping api keys in code directly be a security concern?

Reply:
As per Goodstack’s (formerly Percent) [documentation](https://reference.poweredbypercent.com/#section/Authentication), the publishable key prefixed with **pk_** is intended for safe client-side usage in JavaScript. This key is used solely to identify our account and does not expose any sensitive information. Therefore, when this block was built, it was not necessary to hide this identifier.